### PR TITLE
Shreya

### PR DESCRIPTION
**Pull request by shreyarastogi**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/language/c/loop/While.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//While loop is another loop in C language.
+//It is an entry controlled loop i.e. a test condition is checked and only if it evaluates to true, is the body of the loop executed. 
+
+#include<stdio.h>
```
Comment (arnavb) [inline code/language/c/loop/While.c]: Please follow our C style guide.

File: code/language/c/loop/While.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ int main ()
+{
+     int n = 1,times=5; //local variable initialization
+
+     while( n <= times )//while loops execution
+   {
+      printf("C while loops: %d\n", n);
+      n++;
```
Comment (arnavb) [inline code/language/c/loop/While.c]: Pre-increment.

File: code/language/c/loop/do-while.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//C program to print sum of first 5 natural numbers using do..while loop
+//do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed
+//atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.
+
+#include <stdio.h>
+int main ()
+{
+ int sum = 0, i = 1;  //initialization of counter variable i
+ do{
+   sum = sum+i;
```
Comment (arnavb) [inline code/language/c/loop/do-while.c]: Use `+=`.

File: code/language/c/loop/do-while.c [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+   i++;  //increment of counter variable
+  }while(i <= 5);  //coondition of do while
+
+ printf("sum of first 5 natural numbers = %d",sum);
+ return 0;
+}  //end of program
+
+output:
```
Comment (arnavb) [inline code/language/c/loop/do-while.c]: Place these as comments.

File: code/language/c/loop/While.c (hunk 2 above)
Comment (shreyarastogi) [reply code/language/c/loop/While.c]: done

File: code/language/c/loop/do-while.c [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// do-while loop
+//C program to print sum of first 5 natural numbers using do..while loop
+//do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed
+//atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.
+
+#include <stdio.h>
+int main ()
+{
+ int sum = 0, i = 1;  //initialization of counter variable i
+ do{
```
Comment (Vaibhavi24) [inline code/language/c/loop/do-while.c]: Follow consistent style. Put braces in the next line

File: code/language/c/loop/do-while.c [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+   sum += i;
+   i++;  //increment of counter variable
+  }while(i <= 5);  //coondition of do while
```
Comment (Vaibhavi24) [inline code/language/c/loop/do-while.c]: here too

**Comment by agzuniverse** [comment]
@shreyarastogi do read through the style guide for C - https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c

You can find many code formatters that does the formatting for you, including some websites, plugins to popular text editors, etc.

**Comment by shreyarastogi** [comment]
hey, I have resolved the issues.

File: code/language/c/loop/While.c [hunk 7]
```diff
@@ -0,0 +1,22 @@
+/*While loop is another loop in C language.*/
+/*It is an entry controlled loop i.e. a test condition is checked and only if it evaluates to true, is the body of the loop executed.*/ 
+
+#include<stdio.h>
+
+int main() {
+  int n = 1, times = 5; //local variable initialization
```
Comment (agzuniverse) [inline code/language/c/loop/While.c]: Please use `/*...*/` for comments

**Comment by shreyarastogi** [comment]
Hey! resolved the issues.

File: code/language/c/loop/do-while.c [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+/*do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed*/
+/*atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.*/
+
+#include <stdio.h>
+
+int main() {
+  int sum = 0, i = 1; /*initialization of counter variable i*/
+  do {
+    sum += i;
+    i+=1; /*increment of counter variable*/
```
Comment (Vaibhavi24) [inline code/language/c/loop/do-while.c]: It should be 'i += 1'.
Add the spaces wherever necessary according to our C style guide.

**Comment by aSquare14** [comment]
@shreyarastogi  Please refer which issue your Pull Request fixes in further contributions.

File: code/language/c/loop/do-while.c [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+  } while (i <= 5); /*coondition of do while*/
```
Comment (aSquare14) [inline code/language/c/loop/do-while.c]: typo, fix coondition to condition

File: code/language/c/loop/do-while.c [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+  printf("sum of first 5 natural numbers = %d", sum);
```
Comment (aSquare14) [inline code/language/c/loop/do-while.c]: sum -> Sum

File: code/language/c/loop/While.c [hunk 11]
```diff
@@ -0,0 +1,22 @@
+/*While loop is another loop in C language.*/
```
Comment (aSquare14) [inline code/language/c/loop/While.c]: A while loop in C programming repeatedly executes a target statement as long as a given condition is true. This sounds better than saying it is another loop. Another loop sounds informal.

**Comment by shreyarastogi** [comment]
Made the changes.

File: code/language/c/loop/While.c [hunk 12]
```diff
@@ -0,0 +1,22 @@
+/*A while loop in C programming repeatedly executes a target statement as long as a given condition is true.*/
+/*It is an entry controlled loop i.e. a test condition is checked and only if it evaluates to true, is the body of the loop executed.*/ 
+
+#include<stdio.h>
+
+int main() {
```
Comment (arnavb) [inline code/language/c/loop/While.c]: This doesn't follow our C style guide. Look at all aspects of it.

File: code/language/c/loop/While.c [hunk 13]
```diff
@@ -0,0 +1,22 @@
+/*A while loop in C programming repeatedly executes a target statement as long as a given condition is true.*/
+/*It is an entry controlled loop i.e. a test condition is checked and only if it evaluates to true, is the body of the loop executed.*/ 
+
+#include<stdio.h>
```
Comment (arnavb) [inline code/language/c/loop/While.c]: Add space before `<`.

File: code/language/c/loop/do-while.c [hunk 14]
```diff
@@ -0,0 +1,20 @@
+/* do-while loop*/
+/*C program to print sum of first 5 natural numbers using do..while loop*/
+/*do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed*/
+/*atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.*/
+
+#include <stdio.h>
+
+int main() {
+  int Sum = 0, i = 1; /*initialization of counter variable i*/
```
Comment (arnavb) [inline code/language/c/loop/do-while.c]: Lowercase sum.

File: code/language/c/loop/do-while.c [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+  do {
+    Sum += i;
+    i += 1; /*increment of counter variable*/
```
Comment (arnavb) [inline code/language/c/loop/do-while.c]:
```suggestion
    ++i; /*increment of counter variable*/
```

**Comment by shreyarastogi** [comment]
hey! made the changes.

File: code/language/c/loop/While.c [hunk 16]
```diff
@@ -0,0 +1,23 @@
+/*A while loop in C programming repeatedly executes a target statement as long as a given condition is true.*/
+/*It is an entry controlled loop i.e. a test condition is checked and only if it evaluates to true, is the body of the loop executed.*/ 
+
+#include <stdio.h>
+
+int 
+main()
```
Comment (aSquare14) [inline code/language/c/loop/While.c]: int main () write in one line

File: code/language/c/loop/do-while.c [hunk 17]
```diff
@@ -0,0 +1,22 @@
+/* do-while loop*/
+/*C program to print sum of first 5 natural numbers using do..while loop*/
+/*do- while loop is a loop for performing repetetive tasks. In this loop the body of the loop is executed*/
```
Comment (aSquare14) [inline code/language/c/loop/do-while.c]: repetitive ( typo)

File: code/language/c/loop/do-while.c [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+/*atleast once, and then the test condition is checked. If it evaluates to true, the loop continues to run otherwise it is exited.*/
+
+#include <stdio.h>
+
+int 
+main()
```
Comment (aSquare14) [inline code/language/c/loop/do-while.c]: int main()

**Comment by shreyarastogi** [comment]
Done!

**Comment by agzuniverse** [comment]
@aSquare14 actually, according [the C style guide](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c), `int main()` written on different lines is preferred.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

**Pull request merged**